README.rst

```diff
@@ -51,55 +54,132 @@ Alternatively, if you already have a running Open edX instance, just run the nec
 .. code-block:: bash
 
     tutor dev|local|k8s do init --limit=wordpress
+    tutor dev|local|k8s start wordpress
+
+phpMyAdmin
+**********
+
+The plugin includes optional phpMyAdmin support for managing your WordPress database. To enable it:
+
+1. Set ``RUN_PHPMYADMIN: true`` in your ``config.yml`` by running:
```

Comment:
I would include phpMyAdmin in a separate plugin -- that way, you don't have to bother about `RUN_PHPMYADMIN` in every template. But that's just me.

Reply:
You're right. People with WordPress love their PHPMyAdmin. I'll create another plugin for it.

Reply:
https://github.com/CodeWithEmad/tutor-contrib-phpmyadmin